qtc_packaging/debian_fremantle/control

```diff
@@ -1,4 +1,4 @@
-Source: xbmcremote
```

Comment:
Just had a quick last read through it. Are you sure you want to do all those removals in the control file? Without having tested it, it seems to break packaging.

Seems you'd rather want to replace them with kodimote entries.